client/ayon_core/hosts/houdini/plugins/create/create_redshift_rop.py

```diff
@@ -54,25 +55,40 @@ def create(self, subset_name, instance_data, pre_create_data):
 
         # Set the linked rop to the Redshift ROP
         ipr_rop.parm("linked_rop").set(instance_node.path())
-
         ext = pre_create_data.get("image_format")
-        filepath = "{renders_dir}{subset_name}/{subset_name}.{fmt}".format(
-            renders_dir=hou.text.expandString("$HIP/pyblish/renders/"),
-            subset_name=subset_name,
-            fmt="${aov}.$F4.{ext}".format(aov="AOV", ext=ext)
-        )
+        multi_layered_mode = pre_create_data.get("multi_layered_mode")
 
         ext_format_index = {"exr": 0, "tif": 1, "jpg": 2, "png": 3}
+        multilayer_mode_index = {"No Multi-Layered EXR File": "1",
+                                 "Full Multi-Layered EXR File": "2" }
+
+        if multilayer_mode_index[multi_layered_mode] == "1": 
+            filepath = "{renders_dir}{subset_name}/{subset_name}.{fmt}".format(
+                renders_dir=hou.text.expandString("$HIP/pyblish/renders/"),
+                subset_name=subset_name,
+                fmt="${aov}.$F4.{ext}".format(aov="AOV", ext=ext)
+            )
+            multipart = False
+
+        elif multilayer_mode_index[multi_layered_mode] == "2":
+            filepath = "{renders_dir}{subset_name}/{subset_name}.$F4.{ext}".format(
```

Comment:
It's a bit weird to add the $AOV right after frame range (according to deadline log)
Keep in mind to make sure it aligns with the `instance.data["expectedFiles"]`
```suggestion
            filepath = "{renders_dir}{subset_name}/{subset_name}.$AOV.$F4.{ext}".format(
```

Reply:
Could we revise the logic in this PR because I'm not sure if I got it right or wrong..  

So if the layer mode is set to `No Multi-Layered EXR File`, we should then 
In `create_redshift_rop.py`
- have the `$AOV` token in `RS_outputFileNamePrefix` 
- set `RS_aovMultipart` to `False`
In `collect_redshift_rop.py`
- append beauty product (`RS_outputFileNamePrefix`  + replace `$AOV` with `beauty`) to `render_products` 
- append aov products (`RS_aovCustomPrefix_{i}` + replace `$AOV` with `RS_aovSuffix_{i}` ) to to `render_products` 
- add `render_products` list to `instance.data["files"]` 

and in `Full Multi-Layered EXR File`, we should then
In `create_redshift_rop.py`
- no need to using `$AOV` token in `RS_outputFileNamePrefix` 
- set `RS_aovMultipart` to `True`
In `collect_redshift_rop.py`
- we shouldn't have `beauty` in the beauty product 
- `RS_aovSuffix_{i}`  should be empty
- add  `RS_outputFileNamePrefix`  to `instance.data["files"]`